lib/evss/configuration.rb

```diff
@@ -46,8 +46,8 @@ def connection
         faraday.response :betamocks if mock_enabled?
         faraday.response :snakecase, symbolize: false
         faraday.response :json
-        faraday.use :remove_cookies
-        faraday.adapter :httpclient
+        faraday.use :immutable_headers
+        faraday.adapter Faraday.default_adapter
```

Comment:
:clap:

Reply:
A bit out of the loop - can you remind me the motivation here?

Reply:
@omgitsbillryan the original reason for using HTTPClient was because Net::HTTP capitalizes headers and the VAAFI service does not follow [the spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2) that headers should be case-insensitive and rejects them unless they're all lowercase 👎  

Subbing in Net::HTTP allows us to use the same client across all our services. Plus HTTPClient is a bit less performant and has an issue with its 🍪  manager 😧